ffmpeg/lpms_ffmpeg.c

```diff
@@ -797,13 +797,15 @@ int lpms_transcode(input_params *inp, output_params *params, int nb_outputs)
     }
     ret = open_output(octx, &ictx);
     if (ret < 0) main_err("transcoder: Unable to open output\n");
+    area += params[i].h * params[i].w;
```

Comment:
This is probably OK for now, but could break if we ever do resolution pass-through, because the resolution could change mid-stream.

Reply:
Right.  This will be changed to report on a per-rendition basis *before* merge :-)